ras_party/controllers/account_controller.py

```diff
@@ -583,9 +581,7 @@ def add_new_survey_for_respondent(payload, tran, session):
     disable_iac(enrolment_code, case_id)
 
     if count_enrolment_by_survey_business(survey_id, business_id, session) == 0:
-        casegroup_ids = get_business_survey_casegroups(survey_id, business_id)
-        for case in get_cases_for_casegroups(casegroup_ids, business_id):
-            post_case_event(case['id'], "RESPONDENT_ENROLED", "Respondent enroled")
+        post_case_event(case_id, "RESPONDENT_ENROLED", "Respondent enroled")
```

Comment:
can we just add the arg names here, better to be explicit for this i think

Reply:
:+1: this will make it easier for new people looking at the service to understand what's happening here